acceptance/routes_test.go

```diff
@@ -0,0 +1,75 @@
+// +build acceptance
+
+package acceptance
+
+import (
+	. "github.com/18F/cf-deck/acceptance/util"
+	. "github.com/18F/cf-deck/acceptance/views"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/sclevine/agouti"
+	"net/http/httptest"
+)
+
+var _ = Describe("AppStructure", func() {
+	var (
+		user        User
+		page        *agouti.Page
+		server      *httptest.Server
+		testEnvVars AcceptanceTestEnvVars
+		spaces      Spaces
+		space       Space
+		app         App
+	)
+
+	testEnvVars = AcceptanceTestEnvVars{}
+	testEnvVars.LoadTestEnvVars()
+
+	BeforeEach(func() {
+		// Start a test server
+		server, testEnvVars = startServer()
+
+		// Create a fresh page to navigate.
+		page = createPage()
+
+		// Create user
+		user = StartUserSessionWith(testEnvVars)
+
+		// Log user in
+		user.LoginTo(page)
+	})
+
+	It("Allow the user to create and destroy app routes", func() {
+		By("going to the app page", func() {
+			user.OpenDropdownOfOrgsOn(page)
+			DelayForRendering()
```

Comment:
I think these individual calls have delays already in them. are they needed here too?